docs/upgrade/mongodb_major_upgrades.md

```diff
@@ -0,0 +1,36 @@
+# MongoDB: Major version upgrades
```

Comment:
Other engines support minor version upgrades maybe we should mention it somewhere

Reply:
Done.